src/react/components/pages/projectSettings/projectForm.ui.json

```diff
@@ -9,6 +9,6 @@
         "ui:widget": "connectionPicker"
     },
     "tags": {
-        "ui:widget": "tagsInput"
+        "ui:field": "tagsInput"
```

Comment:
Are you seeing that the outer field template wrapper isn't rendering with the control? Like the label, etc?